src/rdkafka_ssl.c

```diff
@@ -532,6 +532,9 @@ rd_kafka_transport_ssl_io_event (rd_kafka_transport_t *rktrans, int events) {
  * @brief Verify SSL handshake was valid.
  */
 static int rd_kafka_transport_ssl_verify (rd_kafka_transport_t *rktrans) {
+        if (!rktrans->rktrans_rkb->rkb_rk->rk_conf.ssl.enable_verify) {
```

Comment:
put this at line 541, variable definitions must be at the top of the scope.
remove braces {}
add empty line after

Reply:
@edenhill 
> remove braces {}   

librdkafka C++ style guide follows rules from https://google.github.io/styleguide/cppguide.html#Conditionals
so there it is indicated:
`In general, curly braces are not required for single-line statements, but they are allowed if you like them; `
+
it may be worthwhile to connect a tool (linter, formatter) to auto-format a code as specified styleguide, for example https://clang.llvm.org/docs/ClangFormat.html

Reply:
equested changes is resolved by https://github.com/edenhill/librdkafka/pull/2433/commits/347ae4a5e9f1a0d434b276af17a17010994b0c5d

Reply:
seems like brew hangs on travis osx sometimes, ignore it